src/integrationtest/resources/templates/instances.yml

```diff
@@ -92,13 +98,25 @@ Resources:
           stack: !Ref "AWS::StackName"
         ports:
           8080: 9000
+        list-values:
+          - a
+          - b
+          - "|Ref|appVersion"
+          - Ref: appVersion
+          - 2
+          -
+            - a
+            - b
+            - 10
+            - "|Ref|appVersion"
         root: true
         runtime: Docker
         dockercfg:
           'https://my-private-registry':
             auth: "my-secret-string"
             email: "test@example.com"
         environment:
+          SOME_VALUE_FROM_TEXTFILE: "|Ref|textFileParameter"
```

Comment:
Text file?

Reply:
Don't really know - merged in from cfn-sphere. Looks like it's got something to do with this https://github.com/zalando-stups/taupage.